packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -189,6 +189,8 @@ def _get_json_response_template_to_status_codes(
 
 
 def _api_version_validation(builder: OperationType) -> str:
+    if builder.is_overload:
```

Comment:
CI fails for lack of changelog, please run "pnpm change add" then commit it.

Reply:
i could not run `pnpm change add` successfully locally. let me update after i could resolve it.